src/app/specialist/error.js

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import PropTypes from 'prop-types';
+
+export default function Error({ error }) {
+  return (
+    <div className="w-[800]">
+      <h2>Something went wrong!</h2>
+      <pre>{JSON.stringify(error, null, 4)}</pre>
+    </div>
+  );
+}
+
```

Comment:
@watchlar2000 do we need this page? It's similar to root /app/error.js, without this error page the error will just bubble to the root one

Reply:
I think we can keep global-error only. The intention of this particular error page was to make sure we don't forget to display a proper message on the user's screen in case of any server errors/other errors.